app/services/data_exports/data_storage.rb

```diff
@@ -12,7 +12,7 @@ def zip_transcription_files(transcription)
       if transcription.export_files.attached?
         Dir.mktmpdir do |directory_path|
           transcription_folder = download_transcription_files(transcription, directory_path, single_transcription_export: true)
-          yield zip_files(directory_path, transcription_folder)
+          yield zip_files(directory_path, transcription_folder) if block_given?
```

Comment:
The block is used and not new, but I added this `if block_given?` so that I could run this method without passing a block in my tests. Realistically, maybe adding code just for the sake of a test is not necessary, since I could just be passing an empty block in my test.

Reply:
no that's good practice to only yield if the block is optional on the method. I think this is good but it seemed unrelated to the PR change set.